File: Java/Events/Performance.java
package LTD_API;

import java.util.Date;

public class Performance
{
    private int performanceId;
    private Date performanceDate;
    private boolean isUnreserved;
    
    public int getPerformanceId()
    {
        return performanceId;
    }
    public void setPerformanceId(int performanceId)
    {
        this.performanceId = performanceId;
    }

    public Date getPerformanceDate()
    {
        return performanceDate;
    }
    public void setPerformanceDate(Date performanceDate)
    {
        this.performanceDate = performanceDate;
    }

    public boolean getIsUnreserved()
    {
        return isUnreserved;
    }
    public void setIsUnreserved(boolean isUnreserved)
    {
        this.isUnreserved = isUnreserved;
    }
}

File: PHP/Events/Performance.php
<?php
namespace LTD_API;

class Performance
{
    private $performanceId;
    //date format
    private $performanceDate;
    private $isUnreserved;
    
    public function getPerformanceId()
    {
        return $this->performanceId;
    }
    public function setPerformanceId($performanceId)
    {
        $this->performanceId = $performanceId;
    }

    public function getPerformanceDate()
    {
        return $this->performanceDate;
    }
    public function setPerformanceDate($performanceDate)
    {
        $this->performanceDate = $performanceDate;
    }

    public function getIsUnreserved()
    {
        return $this->isUnreserved;
    }
    public function setIsUnreserved($isUnreserved)
    {
        $this->isUnreserved = $isUnreserved;
    }
}

